lms/static/js/instructor_dashboard/remote_gradebook.js

```diff
@@ -219,10 +280,28 @@
         }
 
         InstructorDashboardRemoteGradebook.prototype.onClickTitle = function() {
+            this.clear_display();
+            this.instructor_tasks.task_poller.start();
+            return this.report_downloads.downloads_poller.start();
+        };
+
+        InstructorDashboardRemoteGradebook.prototype.onExit = function() {
+            this.instructor_tasks.task_poller.stop();
+            return this.report_downloads.downloads_poller.stop();
+        };
+
+        InstructorDashboardRemoteGradebook.prototype.clear_display = function() {
             this.$errors.empty();
             this.$results.empty();
+            $('.msg-confirm').css({
+                display: 'none'
+            });
+            return $('.msg-error').css({
+                display: 'none'
+            });
```

Comment:
Near the top of this file there are 2 helper methods called `this.showResults` and `this.showErrors`. Let's create 2 new helper methods called `setTaskMessageVisibility` and `setTaskErrorVisibility` so we don't need to repeat the selectors and `.css()` call a bunch of times. Something like:

```javascript
this.setTaskMessageVisibility = function(visible) {
  $('.msg-confirm').css({display: visible ? 'block' : 'none'});
}
this.setTaskErrorVisibility = function(visible) {
  $('.msg-error').css({display: visible ? 'block' : 'none'});
}
```

Then we get more readable code like this:

```javascript
InstructorDashboardRemoteGradebook.prototype.clear_display = function() {
  this.$errors.empty();
  this.$results.empty();
  this.setTaskMessageVisibility(false);
  this.setTaskErrorVisibility(false);
};
```

Also, I don't think we need to return anything in these functions that show/hide the messages (`return $('.msg-error')...`)